packages/astro/test/astro-minification-html.test.js

```diff
@@ -0,0 +1,42 @@
+import { expect } from 'chai';
+import * as cheerio from 'cheerio';
+import { loadFixture, isWindows } from './test-utils.js';
+
+describe('minification html', () => {
+	describe('in the dev', () => {
+		let fixture;
+		let devServer;
+		const regex = /[\r\n]+/gm;
+		before(async () => {
+			fixture = await loadFixture({
+				root: './fixtures/minification-html/',
+			});
+			devServer = await fixture.startDevServer();
+		});
+
+		after(async () => {
+			devServer.stop();
+		});
+
+		it('Verify that the HTML code is compressed in the dev', async () => {
+			let res = await fixture.fetch(`/`);
+			expect(res.status).to.equal(200);
+			const html = await res.text();
+			expect(regex.test(html.slice(-100))).to.equal(false);
+		});
+
+	})
+	describe('build', () => {
+		let fixture;
+		const regex = /[\r\n]+/gm;
+		before(async () => {
+			fixture = await loadFixture({ root: './fixtures/minification-html/' });
+			await fixture.build();
+		});
+
+		it('Verify that the HTML code is compressed in the pro', async () => {
+			const html = await fixture.readFile('/index.html');
+			expect(regex.test(html.slice(20))).to.equal(false);
```

Comment:
Could you explain to me how the assertion works? What's `20`? I would find more understandable the followings:
- assign `20` to a variable with a good name, so I understand what's that about
- update the description of the test that explains the assertion
- change the assertion to match the **expected text** instead

Reply:
I don't know what we are testing here. It's a genuine question; I don't understand! 😅

Reply:
I agree with what you said.
I will get a good name for that 20.
thanks for you review

Reply:
Explained offline, but what is happening is that doctype is on a separate line, so the output HTML is like this:

```html
<!doctype html>
<html> ..... 
```

So this is grabbing everything after the doctype and verifying it's all just one line. I'll clean up the tests and add some comments so it's more obvious.